entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -19,6 +19,414 @@ EF7 is also available as [daily builds](https://github.com/dotnet/efcore/blob/ma
+| 5   | Brice Lambson    | {<br/>&nbsp;&nbsp;"Phone":"01632 12349",<br/>&nbsp;&nbsp;"Address": {<br/>&nbsp;&nbsp;&nbsp;&nbsp;"City":"Chigley",<br/>&nbsp;&nbsp;&nbsp;&nbsp;"Country":"UK",<br/>&nbsp;&nbsp;&nbsp;&nbsp;"Postcode":"CH1 5ZH",<br/>&nbsp;&nbsp;&nbsp;&nbsp;"Street":"4 Main St"<br/>&nbsp;&nbsp;}<br/>}          |
+
+> [!TIP]
+> This use of aggregates is very similar to the way JSON documents are mapped when using the EF Core provider for Azure Cosmos DB. JSON columns bring the capabilities of using EF Core against document databases to documents embedded in a relational database.
+
+The JSON documents shown above are very simple, but this mapping capability can also be used with more complex document structures. For example, consider an aggregate type used to represent metadata about a post:
+
+<!--
+public class PostMetadata
+{
+    public PostMetadata(int views)
+    {
+        Views = views;
+    }
+
+    public int Views { get; set; }
+    public List<SearchTerm> TopSearches { get; } = new();
+    public List<Visits> TopGeographies { get; } = new();
+    public List<PostUpdate> Updates { get; } = new();
+}
+
+public class SearchTerm
+{
+    public SearchTerm(string term, int count)
+    {
+        Term = term;
+        Count = count;
+    }
+
+    public string Term { get; private set; }
+    public int Count { get; private set; }
+}
+
+public class Visits
+{
+    public Visits(double latitude, double longitude, int count)
+    {
+        Latitude = latitude;
+        Longitude = longitude;
+        Count = count;
+    }
+
+    public double Latitude { get; private set; }
+    public double Longitude { get; private set; }
+    public int Count { get; private set; }
+    public List<string>? Browsers { get; set; }
+}
+
+public class PostUpdate
+{
+    public PostUpdate(IPAddress postedFrom, DateTime updatedOn)
+    {
+        PostedFrom = postedFrom;
+        UpdatedOn = updatedOn;
+    }
+
+    public IPAddress PostedFrom { get; private set; }
+    public string? UpdatedBy { get; init; }
+    public DateTime UpdatedOn { get; private set; }
+    public List<Commit> Commits { get; } = new();
+}
+
+public class Commit
+{
+    public Commit(DateTime committedOn, string comment)
+    {
+        CommittedOn = committedOn;
+        Comment = comment;
+    }
+
+    public DateTime CommittedOn { get; private set; }
+    public string Comment { get; set; }
+}
+-->
+[!code-csharp[PostMetadataAggregate](../../../../samples/core/Miscellaneous/NewInEFCore7/BlogsContext.cs?name=PostMetadataAggregate)]
+
+This aggregate type contains several nested types and collections. Calls to `OwnsOne` and `OwnsMany` are used to map this aggregate type:
+
+<!--
+        modelBuilder.Entity<Post>().OwnsOne(
+            post => post.Metadata, ownedNavigationBuilder =>
+            {
+                ownedNavigationBuilder.ToJson();
```

Comment:
might be worth mentioning that you only need to call ToJson on the top level. It's been said/implied in the ContactDetails example above, but that one didn't have any children, here it's more visible.

Reply:
Will do in the next PR.